behaviors/README.md

```diff
@@ -74,6 +74,26 @@ The mode of the editor sets syntax highlighting, linting, and other functionalit
 
 _Note:_ We will add more supported modes as we find use cases for them. See [this link](http://codemirror.net/mode/) for the full list of modes supported in codemirror.
 
+# complex-list-item
+
+A component which represents a single item in the `complex-list` component. Functionality is derived from its parent. Behaves similar to a field, as it transcludes other behaviors via the same "slot" mechanism that field uses.
+
+# complex-list
+
+An array of objects with arbitrary properties. Each property may have any behaviors a field is allowed to have, including custom behaviors. Complex-list is similar to Angular's _transcluded directives_ or Advanced Custom Fields' _repeater field_, in that each item in the list is treated like a separate field. Items may also have `_label`, but may not have `_display` or `_placeholder`.
```

Comment:
I think it would be helpful for there to be links to the Angular docs about transclusion or ACF's docs about the repeater field

Reply:
Good idea